content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -16,34 +16,75 @@ in that there is no corresponding InfluxDB OSS release.
 (InfluxDB 1.8.x will continue to receive maintenance updates.)
 
 ### Features
+- Upgrade to Go 1.15.10.
 - Support user-defined *node labels*.
   Node labels let you assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
 - Improve performance of `SHOW SERIES CARDINALITY` and `SHOW SERIES CARDINALITY from <measurement>` InfluxQL queries.
   These queries now return a `cardinality estimation` column header where before they returned `count`.
-- Improve diagnostics for license problems. Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
+- Improve diagnostics for license problems.
+  Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
 - Add improved [ingress metrics](/enterprise_influxdb/v1.9/administration/config-data-nodes/#ingress-metric-by-measurement-enabled--false) to track points written by measurement and by login.
-- Support authorization for Kapacitor via LDAP.
+  Allow for collection of statistics of points, values, and new series written per measurement and by login.
+  This data is collected and exposed at the data node level.
+  With these metrics you can, for example:
+  aggregate the write requests across the entire cluster,
+  monitor the growth of series within a measurement,
+  and track what user credentials are being used to write data.
+- Support authentication for Kapacitor via LDAP.
 - Support for [configuring Flux query resource usage](/enterprise_influxdb/v1.9/administration/config-data-nodes/#flux-controller) (concurrency, memory, etc.).
 - Upgrade to [Flux v0.113.0](/influxdb/cloud/reference/release-notes/flux/#v01130-2021-04-21).
+- Update Prometheus remote protocol. <!-- https://github.com/influxdata/influxdb/pull/17814 -->
```

Comment:
read, write or both??

Reply:
@jacobmarble Can you shed some light on this? Here's the relevant PR: https://github.com/influxdata/influxdb/pull/17814.

Reply:
looks like Read....

Reply:
It updates the Prometheus dependency, in order to add streamed reads. Not a feature I'm familiar with, but I tested the regular /metrics endpoint to ensure that the Prometheus exposition data looked sane.